modules/localgov_paragraphs_layout/config/install/field.field.paragraph.localgov_page_section.lgms_width_of_section.yml

```diff
@@ -0,0 +1,20 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.paragraph.lgms_width_of_section
+    - paragraphs.paragraphs_type.localgov_page_section
+  module:
+    - options
+id: paragraph.localgov_page_section.lgms_width_of_section
+field_name: lgms_width_of_section
+entity_type: paragraph
+bundle: localgov_page_section
+label: 'Width of Section'
+description: ''
+required: false
+translatable: false
+default_value: {  }
+default_value_callback: ''
+settings: {  }
+field_type: list_string
```

Comment:
I don't think the `lgms_width_of_section` field config should be here. This is specific to Microsites. If this would be useful to have in all places Layout Paragraphs are used it should be renamed to `localgov_width_of_section`.